common/character_interactions/00_alliance.txt

```diff
@@ -97,6 +97,11 @@
 				}
```

Comment:
No localization.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/117258701-4dc75500-ae5e-11eb-8c6e-93f5d3705bce.png)

</details>

Moreover, I shouldn't see this message because I just got the event telling me it's succeded. I don't need to see the same info twice.